### ISSUE TITLE
[FEAT] Update UI of Signup and Login Pages to Inherit New Design from Index Page

### ISSUE THREAD

**Issue by Vimall03** [Vimall03/Alimento#149]
**Description:**
The current design of the signup and login pages does not match the newly updated design of the index.php page. To ensure consistency across the website, the UI of the signup and login pages should be updated to reflect the latest design standards seen on the index page.

**Tasks:**
Update the layout and styling of the signup page (signup.php) to match the design of index.php.
Update the layout and styling of the login page (login.php) to follow the same design guidelines.
Ensure responsiveness of both pages across all devices (mobile, tablet, desktop).
Test for cross-browser compatibility (Chrome, Firefox, Safari, Edge).

**Additional Info:**
Screenshots of the current signup page, login page, and the new `index.php` page design are attached for reference.

**Attachments:**

**[Screenshot] Signup Page**
![image](https://github.com/user-attachments/assets/44043866-3e8d-4d20-97b0-387c6f661ef4)

**[Screenshot] Login Page**
![image](https://github.com/user-attachments/assets/31cd7fcc-38be-495d-b0c4-ec68d11624a3)

**[Screenshot] Index Page**
![image](https://github.com/user-attachments/assets/c4dd5e72-63cd-4a3b-9750-b871215b6336)

**Expected Outcome:**
The signup and login pages should visually align with the design of the `index.php` page, providing a seamless user experience across all pages of the site.

Comment under this issue if you are interested in contributing.

**Comment by PragnyaRavikumar** [comment]
I would like to work on this issue, please assign it to me.

**Comment by Vimall03** [comment]
@PragnyaRavikumar, Please share me a Figma design that you are planing to implement.

**Comment by KanishkRaj-RAKA** [comment]
Hey  please assign me this task

**Comment by KanishkRajTech** [comment]
Hey, I want to work on this task. please assign this tasK

**Comment by KanishkRaj-RAKA** [comment]
Thanks for assigning this task to me.

**Comment by Sourabh782** [comment]
if this issue's open can i work on it?

**Comment by Vimall03** [comment]
@Sourabh782 Can you share me a design ?

**Comment by Sourabh782** [comment]
> @Sourabh782 Can you share me a design ?

I an thingking of implementing something same as in #112  something similar to this one?

**Comment by Vimall03** [comment]
Okay, I understand. But please know that I wont be able to accept the PR if the design is not good. So the level will be based on the PR. Is that okay ?

**Comment by Sourabh782** [comment]
hii @Vimall03 sorry for late update but i found some designs :

![Screenshot 2024-10-27 210941](https://github.com/user-attachments/assets/a490ecc4-c81b-483b-baf4-906960337a79)

![Screenshot 2024-10-27 210616](https://github.com/user-attachments/assets/15fb621b-edac-43e5-9d02-6d5502b82105)

![Screenshot 2024-10-27 210526](https://github.com/user-attachments/assets/4aaa18b4-c2c8-4fed-9fb3-ae3d90ea3406)

**Comment by Vimall03** [comment]
Hey @Sourabh782 the design in the second image looks good, Please go ahead with it

**Comment by Sourabh782** [comment]
> Hey @Sourabh782 the design in the second image looks good, Please go ahead with it

Ok sure, working on it👍👍

**Comment by Sourabh782** [comment]
hey @Vimall03 this looks good? 

![Uploading image.png…]()

**Comment by Vimall03** [comment]
Yes, looks good, I believe you have changed the UI of the signup page as well

**Comment by Sourabh782** [comment]
> Yes, looks good, I believe you have changed the UI of the signup page as well

Yess just working on it, i tought first taking your opinion before starting so.

any suggestions on it? cause signup and login pages will be nearly similar layout wise so, it would be easier to make change on login and then starting signup

**Comment by Vimall03** [comment]
Okay, This looks good. Just make sure all the CRED operations works as expected

**Comment by Sourabh782** [comment]
> Okay, This looks good. Just make sure all the CRUD operations works as expected

ys its working

**Comment by Sourabh782** [comment]
hey @Vimall03, while testing i npticed the email verification page for user is also old, should i improve it with this pr? or create a new issue for it?

**Comment by Vimall03** [comment]
Yes, Please update the Email verification page in this PR itsef.

**Comment by Sourabh782** [comment]
Hey @Vimall03 check these: 
![Screenshot 2024-10-29 133559](https://github.com/user-attachments/assets/15535ecf-a6ce-4b6b-b5f5-b16359834f57)
![Screenshot 2024-10-29 132802](https://github.com/user-attachments/assets/e203149f-5be2-4a0e-bcb3-2ffa97becb75)
![Screenshot 2024-10-29 132748](https://github.com/user-attachments/assets/bfc62c84-6eb4-431b-8cce-56b2bdb846cd)

all are working

**Comment by Vimall03** [comment]
Great, Looks good 👍
